module/epochs/machine_account.go

```diff
@@ -41,7 +41,7 @@ func init() {
 	if err != nil {
 		panic(fmt.Errorf("could not convert soft min balance for LN: %w", err))
 	}
-	HardMinBalanceLN, err = cadence.NewUFix64("0.001")
+	HardMinBalanceLN, err = cadence.NewUFix64("0.002")
```

Comment:
Not related to this change, but shouldn't we increase `SoftMinBalanceSN` to cover the upper bound of DKG broadcasts?

Reply:
It should already cover worst-case upper bound for DKG, see https://www.notion.so/dapperlabs/Machine-Account-f3c293593ea442a39614fcebf705a132#1e3b627ce81c4d4195d99c67a1958ebb

Reply:
💡 Nice, yes that's what I meant!